packages/sku/config/webpack/plugins/sku-webpack-plugin/index.js

```diff
@@ -65,7 +63,7 @@ class SkuWebpackPlugin {
         include: this.include,
         oneOf: [
           {
-            compiler: /(vanilla-extract|treat-webpack-loader)/,
+            compiler: /(vanilla-extract)/,
```

Comment:
```suggestion
            compiler: /\bvanilla-extract\b/,
```

Reply:
Do we even need the brackets anymore, since there's only one variant?

Reply:
I replaced the brackets with `\b`s

Reply:
Oh, I though `\b` was brackets, it's word boundary 🤦